.env.sample

```diff
@@ -1,4 +1,4 @@
-POSTGRES_HOST=POSTGRES_HOST
-POSTGRES_DB=POSTGRES_DB
-POSTGRES_USER=POSTGRES_USER
-POSTGRES_PASSWORD=POSTGRES_PASSWORD
\ No newline at end of file
+PGHOST=POSTGRES_HOST
+PGDATABASE=POSTGRES_DB
+PGUSER=POSTGRES_USER
+PGPASSWORD=POSTGRES_PASSWORD
```

Comment:
Why did you change names of env variables?

Reply:
I use neon.tec as PG Database. There was an example how ".env" and "settings.py" should look like:

![Screenshot_2](https://github.com/user-attachments/assets/b80054ab-974f-4cbc-ba40-1e217d0c6a49)